builtin.c

```diff
@@ -474,6 +474,20 @@ XS(XS_builtin_indexed)
     XSRETURN(retcount);
 }
 
+XS(XS_builtin_load_module);
+XS(XS_builtin_load_module)
+{
+    dXSARGS;
+    if (items != 1)
+        croak_xs_usage(cv, "arg");
+    SV *module_name = newSVsv(ST(0));
+    if (!SvPOK(module_name)) {
+        SvREFCNT_dec(module_name);
+        croak_xs_usage(cv, "defined string");
+    }
+    load_module(PERL_LOADMOD_NOIMPORT, module_name, NULL, NULL);
+}
```

Comment:
I'd add a comment here specifically pointing out we leave the module name on the stack, so that becomes its return value. It's intentional. Also probably good to put an `XSRETURN(1)` to ensure we return exactly one item even in list context.

Reply:
Thanks! Updated with:

```c
    /* The loaded module's name is left intentionally on the stack for the
     * caller's benefit, and becomes load_module's return value. */
    XSRETURN(1);
```